packages/e2e/lib/e2e.dart

```diff
@@ -36,6 +37,11 @@ class E2EWidgetsFlutterBinding extends LiveTestWidgetsFlutterBinding {
 
   final Completer<bool> _allTestsPassed = Completer<bool>();
 
+  /// Map which stores failure details.
+  ///
+  /// Method names are used as key.
```

Comment:
I'm not familiar with e2e very much. Which "methods" do we use as keys here? I though each failure would correspond to `test('description', () { ... });`, but there are no methods there.

Reply:
done (changed the comment)